tox-ansible.ini

```diff
@@ -0,0 +1,10 @@
+[ansible]
+
+skip =
+    py3.7
+    py3.8
+    2.9
+    2.10
+    2.11
+    2.12
+    2.13
```

Comment:
This is not needed

Reply:
We still have this file in a lot of our collections we can remove it, anyways it is outdated

Reply:
doesn't tox-ansible complain if you don't have this file at all? maybe keep it as a placeholder with skip = "" for skips we may add in the future?

Reply:
that makes sense to me